tests/Microsoft.DotNet.Framework.Docker.Tests/ImageTests.cs

```diff
@@ -59,40 +59,66 @@ private static string GetFilterRegexPattern(string filter)
 
         [Theory]
         [MemberData(nameof(GetVerifyImagesData))]
-        public void VerifyImages(ImageDescriptor imageDescriptor)
+        public void VerifyImagesWithApps(ImageDescriptor imageDescriptor)
+        {
+            VerifyImages(imageDescriptor, "dotnetapp", true);
+        }
+
+        [Theory]
+        [MemberData(nameof(GetVerifyImagesData))]
+        public void VerifyImagesWithWebApps(ImageDescriptor imageDescriptor)
+        {
+            VerifyImages(imageDescriptor, "webapp", false);
+        }
+
+        private void VerifyImages(ImageDescriptor imageDescriptor, string appDescriptor, bool includeRuntime)
         {
             string baseBuildImage = $"{RepoOwner}/dotnet-framework-build:{imageDescriptor.BuildVersion}-{imageDescriptor.OsVariant}";
             VerifyImageExist(baseBuildImage);
 
-            string baseRuntimeImage = $"{RepoOwner}/dotnet-framework:{imageDescriptor.RuntimeVersion}-{imageDescriptor.OsVariant}";
-            VerifyImageExist(baseRuntimeImage);
+            string [] appBuildArgs;
+            if (includeRuntime)
+            {
+                string baseRuntimeImage = $"{RepoOwner}/dotnet-framework:{imageDescriptor.RuntimeVersion}-{imageDescriptor.OsVariant}";
+                VerifyImageExist(baseRuntimeImage);
+                appBuildArgs = new string[]
+                 {
+                        $"BASE_BUILD_IMAGE={baseBuildImage}",
+                        $"BASE_RUNTIME_IMAGE={baseRuntimeImage}",
+                 };
+            }
+            else
+            {
+                appBuildArgs = new string[]
+                 {
+                        $"BASE_BUILD_IMAGE={baseBuildImage}",
```

Comment:
I think it would be best to not have this duplicated.  You could change appBuildArgs to a list and initialize it with the build image.  The includeRuntime case could then add the runtime image.

Reply:
ping

Reply:
This will be addressed in the next PR update.

Reply:
What is WS?  Sorry, I don't see it.  All the changes in ImageTests.cs are intentional.